aspnetcore/migration/22-to-30.md

```diff
@@ -332,7 +336,11 @@ Apps and libraries targeting ASP.NET Core 2.1 or later should remove any direct
 
 ## Json.NET support
 
-As part of the work to [improve the ASP.NET Core shared framework](https://blogs.msdn.microsoft.com/webdev/2018/10/29/a-first-look-at-changes-coming-in-asp-net-core-3-0/), [Json.NET](https://www.newtonsoft.com/json/help/html/Introduction.htm) has been removed from the ASP.NET Core shared framework. Your app may require this reference if it uses `Newtonsoft.Json`-specific feature such as JsonPatch or converters or if it [formats](xref:web-api/advanced/formatting) `Newtonsoft.Json`-specific types.
+As part of the work to [improve the ASP.NET Core shared framework](https://blogs.msdn.microsoft.com/webdev/2018/10/29/a-first-look-at-changes-coming-in-asp-net-core-3-0/), [Json.NET](https://www.newtonsoft.com/json/help/html/Introduction.htm) has been removed from the ASP.NET Core shared framework.
+
+The default for ASP.NET Core is now [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/api/system.text.json?view=netcore-3.0) which is new in .NET Core 3.0. Consider using `System.Text.Json` when possible because it is high-performance and doesn't require an additional library dependency. However, since `System.Text.Json` is new, it might currently be missing features that your app needs.
```

Comment:
```suggestion
The default for ASP.NET Core is now [System.Text.Json](/dotnet/api/system.text.json?view=netcore-3.0), which is new in .NET Core 3.0. Consider using `System.Text.Json` when possible. It's high-performance and doesn't require an additional library dependency. However, since `System.Text.Json` is new, it might currently be missing features that your app needs.
```